week_1/Project/youtube-fe/styles/video.css

```diff
@@ -0,0 +1,56 @@
+.video-grid {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    column-gap: 16px;
+    row-gap: 40px;
+}
+
+.thumbnail-row {
+    margin-bottom: 12px;
+    position: relative;
+}
+
+.thumbnail {
+    width: 100%;
+    border-radius: 15px;
+}
+
+.video-time {
+    background-color: black;
+    color: white;
+    position: absolute;
+    right: 10px;
+    bottom: 10px;
+    font-family: Arial;
+    font-size: 15px;
+    font-weight: bold;
+    padding-left: 4px;
+    padding-right: 4px;
+    padding-top: 4px;
+    padding-top: 4px;
```

Comment:
@codal-kcolaco why there is padding-top givien twice?
```
padding-top: 4px;
    padding-top: 4px;
```

Reply:
@abhi-codal Sorry for that mistake. Will make sure from next time.